command.go

```diff
@@ -0,0 +1,209 @@
+package redis
+
+// A Command represent a Redis command used withing a Request.
+type Command struct {
+	// Cmd is the Redis command that's being sent with this request.
+	Cmd string
+
+	// Args is the list of arguments for the request's command. This field
+	// may be nil for client requests if there are no arguments to send with
+	// the request.
+	//
+	// For server request, Args is never nil, even if there are no values in
+	// the argument list.
+	Args Args
+}
+
+/*
```

Comment:
Still working on the server-side API for reading commands which is why this block of code is commented out.